cluster_tools/cluster_tools/schedulers/pbs.py

```diff
@@ -53,6 +53,29 @@ def format_log_file_name(job_id_with_index, suffix=".stdout"):
     def get_job_id_string(cls):
         return cls.get_current_job_id()
 
+    def handle_kill(self, *args, **kwargs):
```

Comment:
did you test this (pbs) method?

Reply:
No, I only read the (very good!) docs. Do you know where this could be tested?

Reply:
I just looked up how I originally tested the PBS support when I integrated it. Apparently, I tested it on a production instance of a client for which this was developed. Theoretically, you could try to set up PBS with docker (like it's done for slurm), but I don't think it's necessary. The PBS support is untested now for a long time and we might need to invest some effort into it regardless of this feature once we decide to use it again. So, I'd defer the effort until it's really needed :)

Reply:
Agreed, thanks for looking up these details!